irksome/stepper.py

```diff
@@ -56,24 +58,46 @@ def __init__(self, F, butcher_tableau, t, dt, u0, bcs=None,
         self.solver = NLVS(problem, solver_parameters=solver_parameters)
 
         if self.num_stages == 1 and self.num_fields == 1:
-            self.ks = (stages,)
+            self.ws = (stages,)
         else:
-            self.ks = stages.split()
+            self.ws = stages.split()
+
+        A1, A2 = splitting(butcher_tableau.A)
+        self.updateb = numpy.linalg.solve(A2.T, butcher_tableau.b)
```

Comment:
A2 may not be invertible so catch `LinAlgError` here and report something useful?

Reply:
`getForm` would have failed if `A2` were not invertible (see above discussion).  I could have `getForm` return the factors, but computing them is cheap and the return value from `getForm` is already pretty complicated.  But I added a try/except around this.